src/main/java/wellnus/command/HelpCommand.java

```diff
@@ -29,8 +30,9 @@ public class HelpCommand extends Command {
     private static final String USAGE_REFLECT = "\tusage: reflect";
     private static final String USAGE_FOCUS = "\tusage: ft";
     private static final String USAGE_GAMIFICATION = "\tusage: gamif";
-    private static final String ERROR_UNKNOWN_COMMAND = "Sorry, we couldn't find that command!\n"
-            + "To find a command accessible in this part of WellNUS++, try `help`!";
+    private static final String ERROR_UNKNOWN_COMMAND = "Invalid command issued!";
+    private static final String COMMAND_INVALID_COMMAND_NOTE = "Please try 'help' command to check the "
+            + "available commands and their usages!";
```

Comment:
Similar comment as the above `HelpCommand`, a bit weird to ask users to call `help` command when this is `help` command?

Reply:
Either we remove this for all `help`(so blank note message) or we update all commands to print the usage syntax as the note message?